ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-main.yaml

```diff
@@ -35,6 +51,13 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: metrics-linter
+      commands: export GOFLAGS= && make lint-metrics
+      from: ssp-metrics-names-linter
```

Comment:
I would change to `ssp-metrics-linter` since you used `metrics-linter` in line 16

Reply:
I would change both lines (:

Reply:
You don't find `metrics-linter` and `ssp-metrics-linter` more confusing than `metrics-linter` and `ssp-metrics-names-linter`?

Reply:
I actually find `metrics-linter` and `ssp-metrics-names-linter` more confusing, are we taking about 2 linters? one is metrics linter and one is metrics names linter?

Reply:
but just an opinion, maybe I'm wrong.

Reply:
Got your point. updating.

Reply:
Done